assignment5/assignment5.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+function readingFile(file) {
```

Comment:
rename this function to `readFile` or something that describe the work that this function does, `readingFile` seems like this function is currently executing and will execute forever. The function name should not be in at least Present Continuous Tense.